metabase/metabase
# Migrate setup to the new database form

Epic https://github.com/metabase/metabase/issues/26178
Blocked by https://github.com/metabase/metabase/pull/26819

How to test:
- Run a fresh metabase instance
- Make sure it is possible to connect your db during setup

<img width="714" alt="Screenshot 2022-11-30 at 13 51 38" src="https://user-images.githubusercontent.com/8542534/204789285-7a1793ac-1a5c-4104-88fe-49e35b645710.png">
